src/traces/image/attributes.js

```diff
@@ -84,13 +84,13 @@ module.exports = extendFlat({
         valType: 'any',
         dflt: 0,
         editType: 'calc+clearAxisTypes',
-        description: 'Set the image\'s x position.'
+        description: 'Set the image\'s x position. The bottom left corner of the image will be found at xmin=x0-dx/2'
     },
     y0: {
         valType: 'any',
         dflt: 0,
         editType: 'calc+clearAxisTypes',
-        description: 'Set the image\'s y position.'
+        description: 'Set the image\'s y position. The bottom left corner of the image will be found at y0-dy/2'
```

Comment:
```suggestion
        description: [
            'Set the image\'s y position. The top edge of the image',
            '(or the bottom edge if the y axis is NOT reversed or if dy is negative)',
            'will be found at ymin=y0-dy/2. By default when an image trace is',
            'included, the y axis will be reversed so that the image is right-side-up,',
            'but you can disable this by setting yaxis.autorange=true or by providing',
            'an explicit y axis range.'
        ].join(' ')
```
@archmoj there might be other places we should describe this y axis default inversion? but at least here it makes sense since we're discussing which corner `y0` controls.

BTW there seem to be some issues with negative `dy` and hover. I'll make an issue...